postman.go

```diff
@@ -171,10 +176,7 @@ func addItemToCollection(url string, collection *postman.Items, op *swagger.Oper
 		})
 	}
 
-	name := op.OperationID
-	if opIDs := strings.Split(op.OperationID, "."); len(opIDs) > 1 {
-		name = opIDs[1]
-	}
+	name := fmt.Sprintf("%s %s", method, url)
```

Comment:
If it would be in our service I would have suggested to use simple string concatenation for better performance but since this is a CLI tool I'm OK with that. (Although I see this function is called in a loop 🙈 )

Also: could be inlined directly - similar as other fields (Path, ...).

Reply:
Oh what a great advice. Addressed in https://github.com/zalora/bee/pull/38/commits/a0a35451555e78598212803c4a473913535c2ed6